src/modules/0.0.21/iasql_functions/rpcs/iasql_modules_list.ts

```diff
@@ -0,0 +1,33 @@
+import { IasqlFunctions } from '..';
+import { Context, RpcBase, RpcResponseObject } from '../../../interfaces';
+import * as iasql from '../iasql';
+
+export class IasqlModulesList extends RpcBase {
+  module: IasqlFunctions;
+  outputTable = {
+    module_name: 'varchar',
+    module_version: 'varchar',
+    dependencies: 'json',
+  } as const;
+  call = async (
+    dbId: string,
+    _dbUser: string,
+    _ctx: Context,
+  ): Promise<RpcResponseObject<typeof this.outputTable>[]> => {
+    const res = await iasql.modules(true, false, dbId);
+    return res.map(rec => {
+      const updatedRec = {
+        moduleName: rec.moduleName,
+        moduleVersion: `${rec.moduleVersion}`,
```

Comment:
How is this any different from:

```suggestion
        moduleVersion: rec.moduleVersion,
```